requirements/edx-platform-constraints.txt

```diff
@@ -0,0 +1,238 @@
+### DON'T edit this file, it's copied from edx-platform. See make upgrade
```

Comment:
Do we want to commit this, or should it be listed in `.gitignore`?  Seems like it's not strictly needed except when generated during `make upgrade` execution.

Reply:
I figured checking it in would make it easier for named releases, or tracking down problems.